glusterd2/commands/volumes/volume-create.go

```diff
@@ -20,8 +20,8 @@ import (
 )
 
 const (
-	maxMetadataSizeLimit = 4096
-	minVolumeSize        = 20
+	maxMetadataSizeLimit = 4194304
```

Comment:
can you define constants for bytes, KB, MB in some utils folder

and make this as 4 * KB 

its hard to find what is this value, if we add constants it gives more readability.
consider this update in other places too.

Reply:
Ok, will stick to KB for now.